### PR TITLE
Upgrades to Angular 8

### PR DESCRIPTION
**Pull request by aarongranick-okta**
- Requires node >= 12

### CODE REVIEW

File: custom-login/browserslist [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (swiftone) [inline custom-login/browserslist]: Shouldn't we default to match our official supported browsers?

File: okta-hosted-login/browserslist [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (swiftone) [inline okta-hosted-login/browserslist]: Same comment

File: custom-login/package.json [hunk 3]
```diff
@@ -12,33 +12,32 @@
...
-    "@angular/animations": "^7.0.0",
-    "@angular/common": "^7.0.0",
-    "@angular/compiler": "^7.0.0",
-    "@angular/core": "^7.0.0",
-    "@angular/forms": "^7.0.0",
-    "@angular/http": "^7.0.0",
-    "@angular/platform-browser": "^7.0.0",
-    "@angular/platform-browser-dynamic": "^7.0.0",
-    "@angular/router": "^7.0.0",
+    "@angular/animations": "^8.0.0",
```
Comment (swiftone) [inline custom-login/package.json]: The README should update to reflect the supported versions as well (or have that discussion removed)

File: custom-login/browserslist (hunk 1 above)
Comment (aarongranick-okta) [reply custom-login/browserslist]: this was generated by the ng update, but seems good. This would restrict to IE Edge+

Comment (swiftone) [reply custom-login/browserslist]: Per our offline discussion, it makes more sense to not change this until we have testing to ensure the change is accurate.

**Pull request merged**